packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv4/ipv4.controller.js

```diff
@@ -434,7 +452,14 @@ export default class AgoraIpV4OrderController {
   }
 
   manageLoadIpOffers() {
-    this.trackStep(2);
+    if (this.model.selectedRegion) {
+      this.trackClick(
+        `select_region::next_${this.model.selectedRegion.location.replaceAll(
+          ' ',
+          '-',
+        )}`,
+      );
+    }
```

Comment:
this is tracking for select region because we have the select region is last step we dont have button to validate region this is why I put the tracking on order button and some time we order without region this is why we have if check here

Reply:
Ha okay got it its for non vrack and parking products.